near-workspaces/__tests__/mint.ava.ts

```diff
@@ -0,0 +1,70 @@
+import { NearAccount, Workspace } from 'near-workspaces-ava'
+import { CONTRACT_EXTRA, CONTRACT_METADATA } from '../utils/dummyData'
+import {
+    call_mint, view_nft_tokens, view_nft_total_supply,
+} from '../utils/functions'
+
+const workspace = Workspace.init(async ({ root }) => ({
+    contract: await root.createAndDeploy(
+        'cnft',
+        '../build/release/cNFT.wasm',
+        {
+            method: 'init',
+            args: {
+                owner_id: "alice.test.near",
+                contract_metadata: CONTRACT_METADATA,
+                contract_extra: CONTRACT_EXTRA,
+            },
+        }
+    ),
+    alice: await root.createAccount('alice'),
+    john: await root.createAccount('john'),
+    user3: await root.createAccount('user3'),
```

Comment:
Is it possible to actually generate 100 accounts that mint 100 tokens (`userj: await.createAccount('userj')`)

Reply:
Hey @devey, it slows down the test a bit but it's done